e2e/test/iothub/device/TwinE2ETests.cs

```diff
@@ -294,6 +299,42 @@ public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet_Am
                 .ConfigureAwait(false);
         }
 
+        [TestMethod]
+        [Timeout(TestTimeoutMilliseconds)]
+        public async Task Twin_ServiceSetsDesiredPropertyAndDeviceReceivesItOnNextGet_DateTimeProperties_Mqtt()
```

Comment:
The serailization behavior is independent of the transport protocol used, so you should be ok adding test for any one variant and removing the rest. Add in a comment that calls this out and it should be good.